requirements.txt

```diff
@@ -27,4 +27,5 @@ ubuntu-release-info==21.1
 launchpadlib==1.10.13
 macaroonbakery==1.3.1
 sortedcontainers==2.3.0
-vcrpy-unittest==0.1.7
\ No newline at end of file
+vcrpy-unittest==0.1.7
+pillow
```

Comment:
Do we need this?

Reply:
it probably needs to be pinned to a version, but this is the PIL package that draws (I think)

Reply:
yes this should be pinned - I imagine renovate might come round and pin it for you if you don't ;)